templates/elasticsearch.yaml.j2

```diff
@@ -0,0 +1,124 @@
+apiVersion: v1
```

Comment:
I bet we could move this down next to the other service again. I'm not sure moving it up here had any effect.